src/services/RejectDialogService.js

```diff
@@ -0,0 +1,62 @@
+const Dialog = require("../models/Dialog");
+const Response = require("../utils/responses");
+
+module.exports = async (req, res) => {
+  const response = new Response(res);
+  const { entities } = response;
+
+  const { dialogId } = req.params;
+  const { userId } = req;
+
+  try {
+    const dialogue = await Dialog.findById(dialogId);
+
+    if (!dialogue)
+      return response
+        .isError()
+        .entity(entities.DIALOG)
+        .code(response.NOT_FOUND_404)
+        .message("Resource not found")
+        .send();
+
+    const index = dialogue.approvals.indexOf(userId);
+    if (index > -1) {
```

Comment:
```suggestion
  if (index > -1) dialogue.approvals = dialogue.approvals.splice(index, 1);
```

Reply:
you can also do this at ApprovalDialogService: https://github.com/barbosamaatheus/english-talking-api/blob/develop/src/services/ApprovalDialogService.js